### PR TITLE
Added Health Check Implementation for Spruce Controller.

### PR DESCRIPTION
**Pull request by skt123**
@jackchi @ShunmugaSundar 
Added Health check for Spruce Controller with checkInterval of 12 mins.
Please review and merge.

### CODE REVIEW

File: devicetypes/plaidsystems/spruce-controller.src/spruce-controller.groovy [hunk 1]
```diff
@@ -458,7 +469,7 @@ def configure() {
         "zcl global send-me-a-report 0x09 0x00 0x21 1 0 {00}", "delay 500",
         "send 0x${device.deviceNetworkId} 1 1", "delay 500"
 	]
-    return configCmds + rain() + manual()
+    return configCmds + zigbee.onOffConfig(0, 300) + rain() + manual()
```
Comment (jackchi) [inline devicetypes/plaidsystems/spruce-controller.src/spruce-controller.groovy]: Let's use default `onOffConfig` reporting interval

**Review by jackchi — CHANGES_REQUESTED** [review]

**Comment by jackchi** [comment]
Please make the following changes:

1. Change to default onOffConfig()
2. Implement ping with refresh
3. Remove 2 different checkInterval in parse()
4. checkInterval based on default onOffConfig() = 10 *2 + 2= 22min

**Comment by skt123** [comment]
@jackchi 
Made the changes. Please check

**Review by jackchi — APPROVED** [review]
LGTM! 👍

**Pull request merged**